README.md

```diff
@@ -10,10 +10,5 @@ Note: Refresh page if app doesn't load correctly the first time.
 
 ## NEW FEATURES TO ADD:	
 - edit customer name freely for redeemed certificate
-- total revenues (with and without promo)
```

Comment:
I see you included the total revenues less discount, but also created a column for rev + discount. Why is that? 
One should be overall total as if there was no discount, and the other is simply a sum of actual revenue (less discount).